contrib/buildsystems/CMakeLists.txt

```diff
@@ -221,18 +221,22 @@ if(WIN32 AND NOT MSVC)#not required for visual studio builds
 	endif()
 endif()
 
-if(NO_GETTEXT)
```

Comment:
Can you please keep support for `NO_GETTEXT`? It is rather crucial e.g. in the CI builds that we save us a lot of time by _not_ building with translations _every single time_.